### DISCUSSION TITLE
Intune App Update Management

### DISCUSSION THREAD

**Discussion by brandonall** [svrooij/WinTuner#48] [Q&A]
How do you currently manage updates to apps that you have in intune? We run a remediation script to keep apps on devices current, but how do you manage the installers that you upload to intune. Are you just publishing new app versions when they become available?

**Comment by svrooij** [comment]
Hi @brandonall

The latest version of the PowerShell module has support for [Get-WtWin32Apps](https://wintuner.app/docs/wintuner-powershell/Get-WtWin32Apps), those can be repackaged and deployed with [supersede](https://wintuner.app/docs/wintuner-powershell/Get-WtWin32Apps#update-all-and-supersede-the-old-ones).

What this does is creating a new version of the app in Intune and transferring the assignments over. This does not handle the update side on the client, if the apps were not assigned as required.
 
For the question "how do we update apps that are assigned as **available** on the client-side we are open for suggestions. Do you have any ideas?

**Reply by brandonall** [reply]
currently we are running a remediation script to ask winget if there is an update and if so prompt to update. It is not a very elegant solution.

**Reply by svrooij** [reply]
As off version `0.8.4` WinTuner will also set the Auto update part in Intune. So if you previously assigned an app as available and there is an app that supersedes it, Intune will update the app automatically.
